packages/client/src/app/components/form-controls/control-types/options/options.tsx

```diff
@@ -42,9 +45,16 @@ const Options: React.FC<ControlType<OptionsProperty, Field>> = ({
     );
   };
 
+  const convertToCustomValues = (): void =>
+    updateValue({
+      source: Source.Custom,
+      useCustomValues: true,
+      options: options.map((option: Option) => option),
```

Comment:
This still returns the actual references unfortunately,
Better do this

```suggestion
      options: [...options],
```

Reply:
Just wanted to test you... ;)

Reply:
@gustavs-gutmanis Are you sure? Typescript not happy

 ```
TS2322: Type '(Option | OptionGroup)[]' is not assignable to type 'Option[]'.
  Type 'Option | OptionGroup' is not assignable to type 'Option'.
    Property 'value' is missing in type 'OptionGroup' but required in type 'Option'.

```